scripts/globals/items/chocobo_shirt.lua

```diff
@@ -0,0 +1,38 @@
+-----------------------------------------
+--    ID: 10293
+--    Chocobo Shirt
+--    Dispense: Crystals
+-----------------------------------------
+
+-----------------------------------------
+-- OnItemCheck
+-----------------------------------------
+
+function onItemCheck(target)
+    local result = 0;
+    if (target:getFreeSlotsCount() == 0) then
+        result = 308;
+    end
+    return result;
+end;
+
+-----------------------------------------
+-- OnItemUse
+-----------------------------------------
+
+function onItemUse(target)
+    local day = VanadielDayElement()+1;
+    local crystal =
+    {
+        4096, -- Fire
+        4099, -- Earth
+        4101, -- Water
+        4098, -- Wind
+        4097, -- Ice
+        4100, -- Thunder
+        4102, -- Light
+        4103  -- Dark
+    };
+    print(crystal[day]);
```

Comment:
Yeah, leaving it until I can test. I will remove tomorrow if a test tonight goes okay.